packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Live/LiveNow/User.tsx

```diff
@@ -71,7 +71,7 @@ export const User: FunctionComponent<Props> = ({ liveRole, user }) => {
 
       <Stack align="center" className="live-actions" gap={2}>
         {canChangeEditors ? (
-          <>
+          <div>
```

Comment:
Not sure if this change is better? Now we are adding a wrapping `div`, but previously it only renders exactly what it needs? Could also change it to: `{canChangeEditors && liveRole === 'spectator' ? ...}` and then next line: `{canChangeEditors && liveRole === 'editor' ? ...}`, makes it even more explicit I think.

Reply:
Moved to a new small component at the top :)

Reply:
Looks great! 👍  All good from me!